lib/texting_twilio.rb

```diff
@@ -0,0 +1,29 @@
+require 'twilio-ruby'
+require 'dotenv'
+Dotenv.load
+
+module TextingTwilio
+
+  def self.send_text
+
+    account_sid = ENV['ACCOUNT_SID']
+    auth_token = ENV['AUTH_TOKEN']
+    from = ENV['TWIL_PHONE_NUMBER']
+    to = ENV['MY_PHONE_NUMBER']
+
+    delivery_time = Time.new + (60*60)
+    message = "Thank you! Your order has been placed and delivery is scheduled for #{delivery_time.strftime('%H:%M')}"
```

Comment:
Metrics/LineLength: Line is too long. [118/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)